src/util/validationMessages.js

```diff
@@ -36,6 +37,21 @@ const getCannotRemoveMethodMadeInConsoleMsg = (removeMethods, existingMethods) =
   return `${msg}.\nConsider using "bam list" to see which methods can be removed.`;
 };
 
+const getInvalidRuntimeMsg = (runtime) => {
+  const totalValidRuntimes = validNodeRuntimes.length;
+  let validNodeRuntimesStr;
+
+  if (totalValidRuntimes === 2) {
```

Comment:
How would `totalValidRuntimes` ever be something other than `1` when you have set `validNodeRuntimes` as an array with one element?

Reply:
@jasonoverby I implemented it this way in anticipation that AWS might one day support more than one node runtime. Updates to Node are [happening frequently enough](https://github.com/nodejs/Release) that related changes to our codebase could get annoying, so I just wanted to make easy as possible; with this in place, we just have to edit that array of `validNodeRuntimes` when there is an update. 

Now that I look at it again, it might nice to extract this type of information out and into a `settings.js` file.

Reply:
Cool.  I thought you may have been doing that :)